bin/ir_resolver.ml

```diff
@@ -76,10 +76,10 @@ let key k default =
   let doc = Irmin.Private.Conf.doc k in
   let docs = Irmin.Private.Conf.docs k in
   let docv = Irmin.Private.Conf.docv k in
-  let conv = pconv (Irmin.Private.Conf.conv k) in
+  let mk = pconv (Irmin.Private.Conf.conv k) in
```

Comment:
but as a quick fix it is ok, right? I am just trying to make a quick point release to have compat with 0.9 and 1.0 cmdliner.